lib/jxl/dec_frame.h

```diff
@@ -172,73 +172,48 @@ class FrameDecoder {
                                          : std::numeric_limits<size_t>::max());
   }
 
-  void MaybeSetUnpremultiplyAlpha(bool unpremul_alpha) {
-    const jxl::ExtraChannelInfo* alpha =
-        decoded_->metadata()->Find(jxl::ExtraChannel::kAlpha);
-    if (alpha && alpha->alpha_associated && unpremul_alpha) {
-      dec_state_->unpremul_alpha = true;
-    }
-  }
-
-  // Sets the buffer to which uint8 sRGB pixels will be decoded. This is not
-  // supported for all images. If it succeeds, HasRGBBuffer() will return true.
+  // Sets the pixel callback or image buffer where the pixels will be decoded.
+  // This is not supported for all images. If it succeeds, HasRGBBuffer() will
```

Comment:
I believe this comment is stale :)

Reply:
I wanted to update it only after extra channels are also handled and then I can remove HasRGBBuffer() and also JxlDecoderOutputImage()